ansible/playbooks/do_setup.yml

```diff
@@ -0,0 +1,242 @@
+---
+# this playbook will set up a kubernetes cluster, a mongodb database, create a spaces instance to run browsertrix cloud
+# It is assumed that you have a digital ocean API in your environment and have ACCESS keys for your Spaces
+# the playbook also uses doctl
+# set your env variables to have `DO_API_TOKEN` `DO_AWS_ACCESS_KEY` and `DO_AWS_SECRET_KEY`
+- name: deploy browsertrix on  digital ocrean
+  hosts: localhost
+  connection: local
+  gather_facts: false
+    #  vars_files:
+    # - "../group_vars/do/vault.yml"
+  vars:
+    - droplet_size: "s-1vcpu-2gb"
+    - droplet_region: "sfo3"
+    - do_project: "{{ my_project | default('btrixcloud-test')}}"
```

Comment:
Where would `my_project`, `my_space_name` come from, if not set via `-e` flag?
Perhaps we just have a base name, and then have `{{ name }}-k8s` and `{{ name }-db` for each of the different components?

Does it make sense to store this in the inventory system somehow as an alternative to -e?